_overviews/FAQ/index.md

```diff
@@ -44,17 +44,15 @@ reasons for wanting to learn Scala.
 Don't sweat the decision too much.  You can't go far wrong either
 way. It isn't that hard to switch later, in either direction.
 
-Scala 2 remains a common and reasonable choice. Most Scala jobs are
-still Scala 2 jobs. Scala 2 also has a somewhat bigger library
-ecosystem, somewhat better tooling support, and somewhat wider
-availability of learning materials.
-
-That said, these gaps are getting narrower every month.
-Plenty of books are available, a rich library ecosystem already
-exists, and there are more and more jobs using Scala 3.  You should strongly
-consider learning Scala 3. It's still fairly new, released in 2021.
-But it's the future, and it's the best version for
+Regardless, you should choose Scala 3 unless you have a specific reason
+to need 2. Scala 3 is the future, and it's the best version for
 falling in love with the language and everything it has to offer.
+Scala 3 has plenty of books, plenty of libraries, and high quality
+tooling.
+
+That said, many Scala jobs are still Scala 2 jobs. In most cases, the
+cause of that is simply inertia, especially at large shops. (But it can
+sometimes be due to availability of specific libraries.)
```

Comment:
This makes Scala shops sound stodgy and cumbersome.

> Many legacy Scala shops need a kick in the pants to upgrade. Since they are eager for young talent, you need only tell them that you're "only comfortable coding in Scala 3" to flip that script.